### PR TITLE
removed the "en" on the link refer to the php.net manual

### PR DESCRIPTION
**Pull request by david-adam**

### CODE REVIEW

File: zh/rules/always-prepare-statement.md [hunk 1]
```diff
@@ -73,5 +73,5 @@ $sqlQuery = "SELECT column FROM table WHERE id = 10";
 -->
 
 ## 进一步阅读
-*[准备称述](http://php.net/manual/en/mysqli.quickstart.prepared-statements.php)
+*[准备称述](http://php.net/manual/mysqli.quickstart.prepared-statements.php)
```
Comment (cxbig) [inline zh/rules/always-prepare-statement.md:77]: Hi, @david-adam 
Please us `zh` instead. We translate them into Chinese. Then we provide Chinese links

Comment (cxbig) [inline zh/rules/always-prepare-statement.md:77]: And, If you wanna remove the language from URL.
You should remove `manual/en/`. Not only `en/`

Comment (david-adam) [inline zh/rules/always-prepare-statement.md:77]: It's automatically redirect to zh version.

http://php.net/manual/mysqli.quickstart.prepared-statements.php =>
http://php.net/manual/zh/mysqli.quickstart.prepared-statements.php

Comment (cxbig) [inline zh/rules/always-prepare-statement.md:77]: That's weird. @David. For me a link with `manual` only will get an empty page sometimes. 

Hi, @Damien. Can you confirm which is the best?

If David confirmed the links without `manual/en` is also working in China.

I will change them.

—
Sincerely
Buck

**Comment by dseguy** [comment]
No language indication is good, because PHP.net will do.

You may provide link to Chinese article, and / or keep the link to other languages if they are relevant. 
Whenever possible, just don't include the language in the link and let the target website negotiate language with the user.

**Pull request merged**